### DISCUSSION TITLE
How to style user location in GeolocateControl

### DISCUSSION THREAD

**Discussion by MiguelSuay** [uber/react-map-gl#1526] [Q&A]
I am trying to customize the blue dot showing user location displayed on GeolocateControl.

I read the documentation but could not find how to style it, or how to display a svg in place of the blue dot.

Any guidance towads what I should be doing would be helpful.

**Comment by jtuvaleva** [comment]
Hey! I have the same problem. Did you find solution?

**Reply by MiguelSuay** [reply]
Not really, for the moment I forked the repository and changed directly the geolocate-control.js file.
In the function "_renderMarker = ()" you just add an svg image to the displayed marker and it works.

So, it works, but not an optimal solution.

**Reply by Cmoen11** [reply]
Uhm, where's the _renderMarker function located? I can't find it?
